### DISCUSSION TITLE
Charge Level Displaying as Blocks

### DISCUSSION THREAD

**Discussion by PlaysWithTrons** [teslamate-org/teslamate#1823] [Getting started]
I am less than a week into using TeslaMate, and I love it. Although I am confused by the charging level dashboard. Why are there blocks instead of lines like in most of the other graphs?
![Charge Level](https://user-images.githubusercontent.com/87242624/125172750-63703800-e189-11eb-8510-7038dbc22caa.jpeg)

**Comment by PlaysWithTrons** [comment]
I can now add that these blocks only display when the timescale is >= 7days. Less than that it appears "correctly."
![Charge Level - Grafana 2day](https://user-images.githubusercontent.com/87242624/125297020-db0ea600-e2f4-11eb-9bb2-5cc2a23c3266.jpg)

**Reply by SteveDonie** [reply]
Chiming in that I see the same behavior, using Teslamate 1.23.7
